admin/timescale/create_views.sql

```diff
@@ -1,18 +1,8 @@
 BEGIN;
 
-CREATE VIEW listen_count
-       WITH (timescaledb.continuous, timescaledb.refresh_lag=43200, timescaledb.refresh_interval=3600)
-         AS SELECT time_bucket(bigint '86400', listened_at) AS listened_at_bucket, user_name, count(listen)
-            FROM listen group by time_bucket(bigint '86400', listened_at), user_name;
-
-CREATE VIEW listened_at_max
-       WITH (timescaledb.continuous, timescaledb.refresh_lag=43200, timescaledb.refresh_interval=3600)
-         AS SELECT time_bucket(bigint '86400', listened_at) AS listened_at_bucket, user_name, max(listened_at) AS max_value
-            FROM listen group by time_bucket(bigint '86400', listened_at), user_name;
-
-CREATE VIEW listened_at_min
-       WITH (timescaledb.continuous, timescaledb.refresh_lag=43200, timescaledb.refresh_interval=3600)
-         AS SELECT time_bucket(bigint '86400', listened_at) AS listened_at_bucket, user_name, min(listened_at) AS min_value
-            FROM listen group by time_bucket(bigint '86400', listened_at), user_name;
+CREATE VIEW listen_count_30day
```

Comment:
Given that we are on Timescale 2 in prod now, we'll probably need to rewrite this query. The breaking changes in creating continuous aggregates are listed at https://docs.timescale.com/timescaledb/latest/overview/release-notes/changes-in-timescaledb-2/#continuous-aggregates.

Reply:
Yep, for sure. I'll try and work on this later today.